spec/mailers/previews/alert_mailer_preview.rb

```diff
@@ -73,4 +85,44 @@ def example_no_med_training_for_course
               article: example_article,
               course: example_course)
   end
+
+  public
+
+  def create_example_survey_response_alert
```

Comment:
This method and `create_answer` show up as preview links at `/rails/mailers`. I'll comment them out, and update the instructions.